src/main/java/uk/gov/hmcts/ecm/common/model/ccd/types/ClaimantIndType.java

```diff
@@ -29,13 +29,19 @@
     private String claimantGender;
 
     public String claimantFullNames() {
-        return String.join(" ", notNullOrEmptyAtt(new ArrayList<>(), Arrays.asList(claimantTitle,
-                claimantTitleOther, claimantFirstNames, claimantLastName)));
+        var fullNameList = Arrays.asList(claimantTitle, claimantFirstNames, claimantLastName);
```

Comment:
Doesn't this belong in the else part of the if statement below?

Reply:
No it doesn't :) FullNameList has a default value, it changes only if the claimantTitle is "Other". I don't think adding else statement will change anything in the way the code works atm.